src/components/Greeting/Greeting.scss

```diff
@@ -0,0 +1,68 @@
+$greetingBg: linear-gradient(
+  to right,
+  rgba(95, 175, 190, 0.8) 0%,
+  rgba(95, 175, 190, 0.6) 100% /*#5fafbe*/
```

Comment:
why was this comment left here?
use full name for greetingBg

Reply:
done